about-us.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>GOLD GYM</title>
+  <!-- Fontawesome cdn -->
+  <script src="https://kit.fontawesome.com/27219c429c.js" crossorigin="anonymous"></script>
+  <!-- Main css style -->
+  <link rel="stylesheet" href="source/style/about.css" />
+</head>
+
+<body>
+  <!-- Navigation Menu Section -->
+  <nav class="nav">
+    <a href="index.html" class="logo"><img src="source/images/logo.svg" alt="GOLD GYM logo"></a>
+    <!-- fa-solid fa-x -->
+    <button id="show-menu"><i class="fas fa-bars" id="change-icon"></i></button>
+    <!-- nav menu -->
+    <ul class="nav-menu" id="nav-menu">
+      <li class="nav-item"><a href="index.html" class="nav-link" id="link-active">Home</a></li>
+      <li class="nav-item"><a href="about-us.html" class="nav-link">About</a></li>
+      <li class="nav-item"><a href="#" class="nav-link">Join</a></li>
+      <li class="nav-item"><a href="#" class="nav-link">Contact</a></li>
+      <li class="nav-item"><a href="#" class="nav-link">News</a></li>
+    </ul>
+  </nav>
+
+  <!-- Hero Section -->
+  <header class="hero-wrapper">
+    <p class="greeting">"Hello Athlete's"</p>
+    <h1 class="hero-header-text">
+      Victory isn't defined by wins and losses, it's defined by effort
+    </h1>
+    <p class="hero-header-desc">Paramount Gym is Monmouth and Middlesex County’s premier fitness facility. Located in
+      Matawan, New Jersey our 6300 square foot facility houses some of the industry’s biggest names.
+    </p>
+    <p class="hero-header-time">Please contact us per Email for any further question about GOLD GYM.</p>
+    <p class="hero-header-time"><a href="#" class="email">hidarahmdi5@gmail.com</a></p>
+  </header>
+
+  <!-- Section Logo Info -->
+  <section class="logo-info">
+    <h1 class="logo-title" id="hide-me">GOLD GYM Logo</h1>
+    <p class="logo-desc">The logo of GOLD GYM was decided through the logo competition from 8.june to 7.july.</p>
+    <div class="logo-img">
+      <img src="source/images/logo.svg" alt="GOLD GYM LOGO">
+    </div>
+  </section>
+
+  <!-- Past Achivment Section -->
+  <section class="achivment">
+    <h1 class="achivment-title">See the past GOLD GYM Achivment's</h1>
+    <p class="achivment-desc">Take a look at the last two GOLD GYM achivment's which took place in USA and BRAZIL.</p>
+    <!-- Achivment 1 -->
+    <div class="achivment-wrapper">
+      <div class="achivment-1 achivment-img">
+        <p class="achivment-year">2022 <span class="achivment-year-desc">Mr. Olympia 2021</span></p>
+      </div>
+      <div class="achivment-2 achivment-img">
+        <p class="achivment-year">2022 <span class="achivment-year-desc">Arnold Classic 2022</span></p>
+      </div>
+    </div>
+  </section>
+
+  <!-- Partener -->
+  <section class="partner">
+    <h1 class="p-header">Partner</h1>
```

Comment:
- [ ] There should be no more than 1 `<h1>` tag on a page. Please take a look at [this article](https://www.contentkingapp.com/academy/headings/) about the headings and fix this issue.